scanner/cmd/scanner/main.go

```diff
@@ -75,6 +76,10 @@ func main() {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
+	if err := continuousprofiling.SetupClient(continuousprofiling.DefaultConfig()); err != nil {
+		zlog.Error(ctx).Err(err).Msg("unable to start continuous profiling")
```

Comment:
Thank you for using `zlog` and following the scanner conventions. Can you move this down to maybe line 90ish, though? The logging is not setup properly for `zlog` until 84. Or, you can also do this here but switch the `zlog` to `golog`. I'm ok with either

Reply:
Good catch! I've pushed the initialization to line 84.